certbot/setup.py

```diff
@@ -88,7 +88,6 @@ def read_file(filename, encoding='utf8'):
     'astroid',
     'mypy',
     'pylint',
-    'typing',  # for python3.4
```

Comment:
I suppose `tools/dev_constaints.txt` can be recalculated since `typing` does not need to be pinned anymore.

Reply:
Nice catch, but it looks like `typing` is still installed because our pinned version of [Sphinx requires it for Python 2](https://github.com/sphinx-doc/sphinx/blob/v1.7.5/setup.py#L38) so I think we should keep the pinning for now.